DesignSafe-CI/portal
# bug/DES-2288: Fix for infinite scroll not triggering

## Overview: ##
For some reason the scroll stops 0.5px away from the trigger point we use to trigger the next data fetch. The fix is to trigger when the user scrolls within 1px of the bottom.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2288](https://jira.tacc.utexas.edu/browse/DES-2288)

## Summary of Changes: ##

## Testing Steps: ##
1. Navigate to a listing with over 100 files and check that infinite scroll works.

## UI Photos:

## Notes: ##
